src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/repositories/AudioPluginRepository.kt

```
@@ -107,6 +107,7 @@ class AudioPluginRepository(
                             return@flatMapCompletable Completable.complete()
                         }
                     }
+                    .onErrorComplete() // complete even if no plugins to initialize
```

Can't we handle this in the flatmap rather than silence all potential errors?